frontend/src/components/Header.tsx

```diff
@@ -53,6 +69,77 @@ const Header = () => {
     }
   }, [isLoggedIn, getUserInfo]);
 
+  useEffect(() => {
+    const getToken = async () => {
+      if (signedAccountId && walletSelector) {
+        try {
+          const keyStore = new nearApi.keyStores.BrowserLocalStorageKeyStore();
+          const { publicKeyBytes, token, signatureBytes, tokenHash } =
+            await createAccessTokenPayload(keyStore, signedAccountId);
```

Comment:
This is good start; I think we will replace with https://github.com/elliotBraem/near-sign-verify tho

We can leave like this; I will fix up with Masa merge